gossip/hub/handlers.go

```diff
@@ -301,6 +301,10 @@ func (h *hubInfo) addSignedBlob(ctx context.Context, apiReq *api.AddSignedBlobRe
 		}
 	}
 
+	// Charge the operation to a per-source quota bucket, so that sources that sign a lot of things
+	// (e.g. Trillian logs with fast STH generation) can't easily swamp the Hub.
+	charge := appendCharge(chargeTo(ctx), apiReq.SourceID)
```

Comment:
One suggestion - it might be worth making this charge be something like `"gossip_source:"+apiReq.SourceID` to avoid the chance of accidental collisions?

Reply:
#41